scraper/profiles.go

```diff
@@ -0,0 +1,43 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package scraper // import "go.opentelemetry.io/collector/scraper"
```

Comment:
We should have this in an `xscraper` package, like we do with [xreceiver](https://pkg.go.dev/go.opentelemetry.io/collector/receiver/xreceiver) for example, so the scraper package can be made stable before profiles are stable too.